mkdocs.yml

```diff
@@ -1,18 +1,16 @@
 site_name: Common Fund Data Ecosystem Documentation
 
-repo_url: https://github.com/nih-cfde/published-documentation
-
 theme:
   name: material
   palette:
-    primary: white
+    primary: indigo
   features:
     - navigation.tabs
     - navigation.instant
-  logo: images/CFDE_Logo-color-web.png
-
-docs_dir: docs
-
+  logo: images/CFDE-logo-white-outline.png
```

Comment:
can you put the logo back? or change it again, the one you put in doesn't render right

Reply:
I reverted back to the original logo. The white outline logo needs to be redone. The original logo doesn't render well in the smaller browser size in the side nav bar.